### DISCUSSION TITLE
Version 1.22.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#3721] [Changelog]
## Bugfixes
- [#3668](https://github.com/ScratchAddons/ScratchAddons/issues/3668) Changelog link in popup cannot be middle-clicked. (prio-4)
- [#3674](https://github.com/ScratchAddons/ScratchAddons/issues/3674) Permissions page does not use light mode screenshot. (prio-4)
- [#3693](https://github.com/ScratchAddons/ScratchAddons/issues/3693) better-emojis does not support Scratch Cat, Gobo and Waffle emojis. (prio-4)
- [#3704](https://github.com/ScratchAddons/ScratchAddons/issues/3704) editor-theme3 reduces the size of clickable area in block dropdowns. (prio-3)
- [#3709](https://github.com/ScratchAddons/ScratchAddons/issues/3709) Addons do not run on Code of Ethics page. (prio-4)
- [#3724](https://github.com/ScratchAddons/ScratchAddons/issues/3724) editor-devtools block context menu can have separators on the top item. (prio-4)

## New Features
- [#3425](https://github.com/ScratchAddons/ScratchAddons/issues/3425) New addon: forum-id (prio-3)
- [#3491](https://github.com/ScratchAddons/ScratchAddons/issues/3491) New addon: columns (prio-3)
- [#3603](https://github.com/ScratchAddons/ScratchAddons/issues/3603) animated-thumb: redesign and add modal (prio-3)
- [#3669](https://github.com/ScratchAddons/ScratchAddons/issues/3669) New addon: search-my-stuff (prio-3)
- [#3701](https://github.com/ScratchAddons/ScratchAddons/issues/3701) New addon: fix-editor-comments (prio-3)
- [#3703](https://github.com/ScratchAddons/ScratchAddons/issues/3703) editor-devtools: add shortcut to not close the find bar (prio-3)
- [#3722](https://github.com/ScratchAddons/ScratchAddons/issues/3722) New addon: swap-local-global (prio-3)
- [#3727](https://github.com/ScratchAddons/ScratchAddons/issues/3727) New addon: editor-comment-previews (prio-3)
- [#3741](https://github.com/ScratchAddons/ScratchAddons/issues/3741) New addon: tutorials-button (prio-3)

**Comment by kccuber-scratch** [comment]
#3546 also got fixed

**Reply by kccuber-scratch** [reply]
nvm it only gets fixed if you turn on `better-emojis`
